service-catalog/components/prow-hook.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: prow-hook
+  title: Hook
+  description: Receives all the web hooks from GitHub. It is a stateless server that listens for GitHub webhooks and dispatches them to the appropriate plugins.
```

Comment:
```suggestion
  description: Receives all the webhooks from GitHub. It is a stateless server that listens for GitHub webhooks and dispatches them to the appropriate plugins.
```

Reply:
this is all taken from upstream.. but :shrug: why not...